torch/csrc/jit/codegen/cuda/dynamic_type.h

```diff
@@ -186,6 +186,14 @@ class TORCH_CUDA_CU_API IntOrDouble {
   explicit operator int64_t() const;
   explicit operator size_t() const;
   explicit operator int() const;
+
+  IntOrDouble abs() const {
```

Comment:
Should we make this a function inside `namespace IntOrDouble_functions` in this file to be consistent with other functions?